dags/monrecap.py

```diff
@@ -38,14 +50,18 @@
         classname="MonRecapBaro",
     )
 
+    @task
+    def drop_barometre_table(**context):
+        if context["params"]["drop_baro"]:
+            con.execute("""drop table if exists monrecap.raw_barometre cascade;""")
```

Comment:
Ça serait mieux de ne pas exécuter la tâche quand le paramètre est présent afin de plus facilement voir quand celle-ci est lancé car là elle sera tout le temps verte.
Je n'ai pas testé mais le plus simple semble être d'utiliser [`@task.skip_if`](https://airflow.apache.org/docs/apache-airflow/2.11.0/_api/airflow/decorators/condition/index.html#airflow.decorators.condition.skip_if) :
```suggestion
    @task.skip_if(lambda params: params["drop_baro"])
    @task
    def drop_barometre_table():
        con.execute("""drop table if exists monrecap.raw_barometre cascade;""")
```
L'alternative que j'ai vu c'est de faire du [branching](https://airflow.apache.org/docs/apache-airflow/2.11.0/core-concepts/dags.html#branching) mais ça devient un peu plus complexe donc peut-être une autre fois :).